dev/devicelab/bin/tasks/run_release_test_macos.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:flutter_devicelab/common.dart';
+import 'package:flutter_devicelab/framework/devices.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/task_result.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+import 'package:path/path.dart' as path;
+
+/// Basic launch test for desktop operating systems.
+void main() {
+  task(() async {
+    deviceOperatingSystem = DeviceOperatingSystem.macos;
```

Comment:
You don't have to do it in this PR but can you also replace this `deviceIdOverride` hack with your nicer `DeviceOperatingSystem.macos` fix? 
https://github.com/flutter/flutter/blob/deacb8d434b4b05291be912da2c4bcac869237cf/dev/devicelab/bin/tasks/hot_mode_dev_cycle_macos_target__benchmark.dart#L9

Reply:
Thanks for spotting this. Looks like we do the same in the Windows/Linux equivalents too. I'll send one patch that does all three and adds the Linux DeviceOperatingSystem if it doesn't exist (the Windows one definitely does).